apps/src/applab/RedirectDialog.jsx

```diff
@@ -69,12 +67,24 @@ export default class RedirectDialog extends React.Component {
     }
 
     return (
-      <Dialog title={title} isOpen={this.state.isOpen}>
-        <Body>
+      <Dialog title={title} isOpen handleClose={this.props.handleClose}>
+        <Body className={'ui-test-redirect-notice'}>
```

Comment:
What does adding this className do?

Reply:
Good catch - thanks! That's a handle I added for a UI test I decided not to implement. Removing . . .